ci/azure-cross-compile.yml

```diff
@@ -1,27 +1,41 @@
 jobs:
 - job: ${{ parameters.name }}
   displayName: ${{ parameters.displayName }}
+  strategy:
+    matrix:
```

Comment:
This matrix is [roughly the same as what we use in bytes.](https://github.com/tokio-rs/bytes/blob/master/ci/azure-cross-compile.yml)